src/ts/UIManager.ts

```diff
@@ -461,7 +461,20 @@ export class UIManager {
     if (onShow) {
       onShow();
     }
+
+    // Switch to new UI variant
     this.currentUi.getUI().show();
+
+    // Trigger synthetic 'mousemove' to ensure UI show/hide logic is initialized.
+    // In UIContainer, we detect user activity with mouse events (e.g. `mousemove`)
+    // and start the auto-hide logic accordingly. However, when switching UI,
+    // the mouse can already be inside the player, so no natural `mousemove` event fires. 
+    const adUiContainer = this.currentUi.getUI();
+    const htmlElement = adUiContainer.getDomElement().get(0);
+    if (htmlElement) {
+      htmlElement.dispatchEvent(new MouseEvent('mousemove', { bubbles: true }));
+    }
```

Comment:
Why did you decide to fake a mouse event insead of just directly calling the same method which is used within the event listener?

```suggestion
    adUiContainer.showUi();
```

Reply:
Update: I just figured out that this is already happening in line 466 above.